src/static/css/infusion-docs.css

```diff
@@ -141,6 +141,11 @@ a:focus {
     font-size: 1.2rem;
 }
 
+/* override the above to avoid paragraphs in tables being resized inappropriately */
+.infusion-docs-container tr.infusion-docs-alt-a-row td p, .infusion-docs-container tr.infusion-docs-alt-b-row td p {
```

Comment:
Not all appropriate table rows are matched by this form - e.g. the first row in the table "Full listener record form". 
I suggest we have a more aggressive rule of something like .infusion-docs-container tr td p

Also, unless the user remembers to add the blank line first, the p does not appear either, leading to a mishmash of sizes as you can see in the "Using an event firer procedurally" table - although hopefully fixing the inherit rule specificity will deal with this

Reply:
I updated as you suggested and couldn't find any oddities in my review.  List content still seems a bit large to me, but I added a row to confirm that pure HTML and markdown lists are identical.

![screen shot 2017-11-06 at 10 40 45](https://user-images.githubusercontent.com/5765222/32434706-2c873ab8-c2df-11e7-852b-23c485ab3bae.png)

I also checked the live site, the style seems to be unchanged from before.  I can adjust that as well if you like.